opal/models.py

```diff
@@ -122,6 +122,52 @@ def _get_field_type(cls, name):
         )
 
     @classmethod
+    def get_human_readable_type(cls, field):
+        field_type = cls._get_field(field)
+
+        if isinstance(field_type, models.BooleanField):
+            return "Either True or False"
+        if isinstance(field_type, models.NullBooleanField):
+            return "Either True, False or None"
+        if isinstance(field_type, models.DateTimeField):
+            return "Date & Time"
+        if isinstance(field_type, models.DateField):
+            return "Date"
+
+        numeric_fields = (
+            models.AutoField,
+            models.BigIntegerField,
+            models.IntegerField,
+            models.FloatField,
+            models.DecimalField,
+        )
+        if isinstance(field_type, numeric_fields):
+            return "Number"
+
+        if isinstance(field_type, ForeignKeyOrFreeText):
+            t = "Any result is possible, but a curated list of {} is suggested"
+            return t.format(field_type.foreign_model._meta.object_name)
```

Comment:
I think this is out of sync with what the front end says in 0.8.2 ?
(In fact I think you've not merged this branch with0.8.2 since the lookuplist modal thing in https://github.com/openhealthcare/opal/pull/1051 )